tests/api-diff/type_tests.t

```diff
@@ -0,0 +1,78 @@
+Here we generate a basic `.mli` file with two types and a function:
+
+  $ cat > ref.mli << EOF
+  > type t = int
+  > type unused_type = string
+  > val f : t -> string
+  > EOF
+
+We generate the .cmi file
+
+  $ ocamlc ref.mli
+
+And now we run api-watcher on that same cmi file as both arguments,
+there should be no diff:
+
+  $ api-diff ref.cmi ref.cmi
```

Comment:
This test is done twice, in `type_tests` and in `value_tests`.

I think it makes sense to extract those "diffs between identical cmis" in a specific file.

Reply:
Looking at the history, you had the tests extracted before! You removed that when adding the `ref.mli` creation to the other tests using the file.

It is okay to create the same `ref.mli` file in different tests. But do bring back your `identical_cmi.t` file!